### PR TITLE
Add expirationDate parameter to user_token

### PR DESCRIPTION
**Pull request by fhusson**
Hi, this Pull Request is to add the ExpirationDate parameter to the user_token ressource.
#113 
Be carefull, I have not tested the code.

### CODE REVIEW

**Comment by jdamata** [comment]
Do you mind adding an acceptance test for this? It should be pretty simple. https://github.com/jdamata/terraform-provider-sonarqube/blob/master/sonarqube/resource_sonarqube_user_token_test.go

**Comment by fhusson** [comment]
> Do you mind adding an acceptance test for this? It should be pretty simple. https://github.com/jdamata/terraform-provider-sonarqube/blob/master/sonarqube/resource_sonarqube_user_token_test.go

Hi sorry it's done, sorry for the delay.

File: sonarqube/resource_sonarqube_user_token_test.go [hunk 1]
```diff
@@ -48,3 +49,35 @@ func TestAccSonarqubeUserTokenBasic(t *testing.T) {
...
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSonarqubeUserTokenExpirationDateConfig(rnd, "testAccSonarqubeUserTokenWithExpirationDate"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(name, "name", "testAccSonarqubeUserTokenWithExpirationDate"),
+				),
```
Comment (jdamata) [inline sonarqube/resource_sonarqube_user_token_test.go:79]: Lets add another Check here for the presence of the token:

```go
resource.TestCheckResourceAttr(name, "expiration_date", "<VALUE>")
```

You'll have to pull the expiry time in testAccSonarqubeUserTokenExpirationDateConfig into a function parameter to reuse that value in  TestAccSonarqubeUserTokenWithExpirationDate.

**Comment by jdamata** [comment]
> > Do you mind adding an acceptance test for this? It should be pretty simple. https://github.com/jdamata/terraform-provider-sonarqube/blob/master/sonarqube/resource_sonarqube_user_token_test.go
> 
> Hi sorry it's done, sorry for the delay.

Thanks!! Just 1 modification to the test case and we should be good to go

**Comment by fhusson** [comment]
> > > Do you mind adding an acceptance test for this? It should be pretty simple. https://github.com/jdamata/terraform-provider-sonarqube/blob/master/sonarqube/resource_sonarqube_user_token_test.go
> > 
> > 
> > Hi sorry it's done, sorry for the delay.
> 
> Thanks!! Just 1 modification to the test case and we should be good to go

Hi it's done :)

File: sonarqube/resource_sonarqube_user_token_test.go (hunk 1 above)
Comment (fhusson) [reply sonarqube/resource_sonarqube_user_token_test.go:81]: Hi it's done :)

**Review by jdamata — APPROVED** [review]

**Pull request merged**